data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,213 @@ mission "Hai Pilgrimage"
+				goto questions
+			label warmonger
+			`	"Anoo was always sympathetic to the cause of the Unfettered Hai. Not their pointless war - she hated that - but she respected their drive to progress and their gift for change. Her father's family tried to stop her ideas, and we thought we succeeded, but... maybe...`
+			`	"A trauma surgeon for the Unfettered Hai? I wonder if she planned this the years in advance?"`
+				goto questions
+			label accept
+			`	"If Anoo went anywhere, it would be the Darkcloak. The largest Unfettered hospital is there. Many soldiers come back from raids injured on Darkcloak. A trauma surgeon would have a lot of work to do."`
+				accept
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+
+mission "Acorn Delights: Part 4"
+	landing
+	name "Acorn Delights"
+	description "Take Eeeya to <destination> to finalize some paperwork regarding the Acorn Delights shop."
+	source "Darkcloak"
+	destination "Allhome"
+	passengers 1
+	to offer
+		has "Acorn Delights: Part 3: done"
+	on offer
+		payment 3000
+		log "Finally found Anoo. It turns out she joined the Unfettered. I am supposed to take Eeeya to <destination> now."
+		conversation
+			`You're ushered from the landing platform to a nearby hospital, where Eeeya finally manages to convey your intent. A few Unfettered Hai nearby are muttering in their language, and you keep hearing the Hai word for "cookies" as they watch Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. Unlike their southern brethren, it seems the Unfettered value function and progress over aesthetics and comfort.`
+			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this hospital is to heal them quickly so that they can fight again as soon as possible.`
+			`	You reach a high floor filled with personal offices. Eeeya knocks on each door, apologizing when the Unfettered inside aren't who she's looking for, until you finally reach the very last room.`
+			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak in Hai, then remember that you are in the room.`
+			`	"Please, Anoo, let's speak in the human language. <first> came all this way for me, and it's only fair that Captain be able to understand what's going on."`
+			`	The conversation then turns to Anoo's new occupation.`
+			`	"Why did you come here, Anoo? The Unfettered hate us."`
+			`	"They don't hate us in particular, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our species is fading into history through a slow death of mediocrity. Most Unfettered see the warfare as our only salvation.`
```

Comment:
Since Anoo's younger and well-educated, I think she should have a good grasp of the human language.
```suggestion
			`	"They don't hate us in particular, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our species is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.`
```